src/checkbox/styles.css

```diff
@@ -7,21 +7,20 @@
   display: inline-flex;
   height : 18px;
   width: 18px;
+  font-size: 12px;
+  color: $dark-gray;
+
   border: 1px solid $light-gray;
-  background-color: $lighter-gray;
+  background-color: white;
```

Comment:
I typically prefer hexcodes rather than native named colors for consistency and reliability